schema.graphql

```diff
@@ -525,10 +548,12 @@ type Mutation {
   unlikePost(id: ID!): Post
   unregisterForEventByUser(id: ID!): Event!
   updateEvent(data: UpdateEventInput, id: ID!): Event!
+  updateEventProject(data: UpdateEventProjectInput!, id: ID!): EventProject!
   updateLanguage(languageCode: String!): User!
   updateOrganization(data: UpdateOrganizationInput, file: String, id: ID!): Organization!
   updatePluginStatus(id: ID!, orgId: ID!): Plugin!
   updatePost(data: PostUpdateInput, id: ID!): Post!
+  updateTask(data: UpdateTaskInput!, id: ID!): Task
```

Comment:
@duplixx 

1. We removed Projects and Tasks from the API a few PRs ago.  Your starting code is out of date
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1627
1. This is going to be replaced with Action Items.
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1648
1. Remove this Type and all references to it